src/main/java/edu/christopherstafford/advancedjava/Run.java

```diff
@@ -0,0 +1,57 @@
+package edu.christopherstafford.advancedjava;
+
+import java.time.LocalDate;
+
+/**
+ *A simple class that models a run workout
+ * 
+ * @author Christopher Stafford
+ */
+public class Run {
```

Comment:
not sure what this class has to do with the current assignment.

Reply:
These are just leftover from the first assignment that seem to have never been merged in.